src/ready/apis/train_mobious.py

```diff
@@ -53,109 +53,7 @@ def main(args):
-
-    device = torch.device("cuda" if torch.cuda.is_available() else "cpu")  
-    device_name = torch.cuda.get_device_name(0)[0:20] if torch.cuda.is_available() else "cpu"
-    device_name= device_name.replace (" ", "_")
-    logger.info(f"GPU DEVICE NAME: {device_name}")
-    cuda_available = torch.cuda.is_available()
-
-    #TOTEST
-    # weight_fn = config.model.weight_fn
-    # logger.info(f"weight_fn: {weight_fn}")
-    # if weight_fn is not None:
-    #     raise NotImplementedError()
-    # else:
-    #     logger.info(f"Starting new checkpoint. {weight_fn}")
-    #     weight_fn = os.path.join(
-    #         os.getcwd(),
-    #         f"checkpoint_{datetime.now().strftime('%Y%m%d_%H%M%S')}.pth.tar",
-    #     )
-
-
-    #TODO degug color transformations
-    # https://pytorch.org/vision/main/auto_examples/transforms/plot_transforms_illustrations.html
-    #TODO calculate the mean and std of the dataset and use them to normalize the images.
-    # https://www.geeksforgeeks.org/how-to-normalize-images-in-pytorch/
-    transforms_img = transforms.Compose([
-                                            transforms.ToImage(),
-                                            transforms.RandomHorizontalFlip(p=0.5),
-                                            transforms.RandomVerticalFlip(p=0.5),
-                                            transforms.RandomRotation(45),
-                                            transforms.GaussianBlur(kernel_size=(5, 13), sigma=(1, 50)),
-                                            transforms.Normalize(mean=[0.285, 0.456, 0.406], std=[0.529, 0.524, 0.525]),
-                                            transforms.ElasticTransform(alpha=100.0, sigma=5.0),
-                                            ])
-    transforms_rotations = transforms.Compose([
-                                            transforms.ToImage(),
-                                            transforms.RandomHorizontalFlip(p=0.5),
-                                            transforms.RandomVerticalFlip(p=0.5),
-                                            transforms.RandomRotation(45),
-                                            ])
-
-    #TODO avoid to hardcode data path in trainset. This should be passed as argument in the confirg file
-    ## Length 5; github_data_path
-    ## Length 1143;  data_path
-    trainset = MobiousDataset(
-        FULL_GITHUG_DATA_PATH, transform=None, target_transform=None
-        # FULL_GITHUG_DATA_PATH, transform=transforms_img, target_transform=None
-        # FULL_GITHUG_DATA_PATH, transform=transforms_rotations, target_transform=transforms_rotations
-        # FULL_GITHUG_DATA_PATH, transform=transforms_img, target_transform=transforms_rotations
-        # FULL_DATA_PATH, transform=None, target_transform=None
-        # FULL_DATA_PATH, transform=transforms_rotations, target_transform=transforms_rotations
-        # FULL_DATA_PATH, transform=transforms_img, target_transform=transforms_rotations
-    )
```

Comment:
Could we keep somewhere in the doc strings the list of transform and target_transform  that have been used and tested with MobiousDataset? This could be also part of the config file as we will train model with different transforms :)